src/components/all-on-one-form-sub-element-hold-no-modal/all-on-one-form-sub-element-hold-no-modal.ts

```diff
@@ -129,6 +128,7 @@ export class AllOnOneFormSubElementHoldNoModalComponent {
 
   addDangerousFault() {
     if (this.dangerous) return;
+    if (this.isManoeuvre() && (this.faultCounter > 0 || this.serious)) return;
```

Comment:
What happens if somebody goes in and removes the line number 130?

Reply:
💥

Reply:
would add another dangerous fault into the redux store I would think

Reply:
ahh cool :D